components/Navigation/index.js

```diff
@@ -3,33 +3,38 @@ import styled from "styled-components";
 import Link from "next/link";
 import { useContext } from "react";
 import { CartContext } from "../../pages/_app";
-import { CiHome } from "react-icons/ci";
-import { CiHeart } from "react-icons/ci";
-import { CiShoppingCart } from "react-icons/ci";
-import { CiUser } from "react-icons/ci";
+import { RiHomeHeartLine } from "react-icons/ri";
+import { RiStarSmileLine } from "react-icons/ri";
+import { RiShoppingCartLine } from "react-icons/ri";
+import { RiUserHeartLine } from "react-icons/ri";
 
 export default function Navigation() {
   const context = useContext(CartContext);
   return (
     <>
       <StyledListItem>
-        <CiHome color="FFFFCB" fontSize="50" />
-        <Link href="/">Home</Link>
+        <Link href="/">
+          <RiHomeHeartLine color="FFFFCB" fontSize="50" />
+        </Link>
```

Comment:
I think there should be an `aria-label` here to make sure the Links are accessible (see the links below as well).